filter_plugins/filter_hosts.py

```diff
@@ -43,9 +43,15 @@ def get_one_not_expelled_instance(hostvars, play_hosts):
     raise Exception('At least one play host should be non-expelled and not stateboard')
 
 
-def get_instance_control_sock(app_name, inventory_hostname, stateboard=False):
-    instance_fullname = get_instance_fullname(app_name, inventory_hostname, stateboard)
-    return '/var/run/tarantool/{}.control'.format(instance_fullname)
+def get_instance_control_sock(instance_vars):
```

Comment:
I suggest changing other filter functions signatures to receive `instance_vars` arguments.